### DISCUSSION TITLE
Freeform table field to element table field integration

### DISCUSSION THREAD

**Discussion by CAIDcreative** [solspace/craft-freeform#1208] [Feature Requests]
Would be really helpful to be able to populate/edit an element table field via the freeform table field. It really feels like it should be able to do this and finding in the documentation "Freeform Table fields cannot be mapped to other Elements, nor can Freeform fields be mapped to Table fields in other elements." was frustrating (and a little ambiguous at first glance).

**Comment by kjmartens** [comment]
Thanks for your feedback @CAIDcreative,

With the latest improvements in Freeform 5, we can likely add support for this in the near future. 🙂
